code/_onclick/hud/robot.dm

```diff
@@ -98,6 +98,13 @@
 	mymob.throw_icon.icon_state = "store"
 	mymob.throw_icon.name = "store"
 	mymob.throw_icon.screen_loc = ui_borg_store
+	
+//Headlamp
+	mymob:lamp_button = new /obj/screen()
```

Comment:
@Markolie Eugh. This should probably just istype the mob and typecast regardless, as should the above parts.